api/projects.types.ts

```diff
@@ -8,6 +8,8 @@ export enum ProjectStatus {
   stable = "stable",
   warning = "warning",
   critical = "critical",
+  error = "error",
+  info = "info",
```

Comment:
Adding these two is correct but I'd remove stable and critical as those don't exist in the API response

Reply:
👍